.github/commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = {extends: ['@commitlint/config-conventional']}
```

Comment:
I think that we need to be diligent in making sure that contributors get a clear message when the linter fails.

We ought to set the `helpUrl` to `https://github.com/kumahq/kuma/blob/master/CONTRIBUTING.md#commit-message-format`, and maybe there's more explanatory text we should consider?